ilc/client/isActiveFactory.js

```diff
@@ -1,4 +1,4 @@
-import {triggerAppChange} from 'single-spa';
+import {triggerAppChange} from './navigationEvents';
```

Comment:
Do we have here events setting up twice?
Look at `ilc/client` line number 1

Reply:
Nope, due to the cache of the imports. Files are executed only once